pkg/cmd/repo/clone/clone.go

```diff
@@ -87,6 +87,17 @@ func cloneRun(opts *CloneOptions) error {
 
 	var repo ghrepo.Interface
 	var protocol string
+	wikipref := false
```

Comment:
This variable would be better named as `wantsWiki` or something like that. We prefer full names for variables